contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,301 @@
+---
+date: 2023-05-08
+title: "In-depth: PostHog vs Hotjar"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-vs-hotjar/posthog-vs-hotjar.jpg
+featuredImageType: full
+author:
+    - lior-neu-ner
+category: General
+tags:
+    - Comparisons
+---
+
+Want to understand the difference between Hotjar and PostHog? Here's the short answer:
+
+1. Hotjar enables you to see how customers are using your product with session replays and heatmaps. It also provides widgets for collecting feedback and tools for scheduling customer interviews. Hotjar is built for marketers and user experience (UX) researchers.
+
+2. PostHog is an all-in-one product improvement platform for product analytics, session recording, feature flags, in-app prompts, and more. PostHog is built for engineers and product teams.
+
+Now it's time for the long answer...
+
+In this guide, we'll cover:
+
+- Hotjar and PostHog [feature comparison](#feature-comparison)
+- [Session replay](#session-replay) capabilities
+- [Privacy](#privacy-compliance) and [security](#admin--security)
+- [Frequently asked questions](#frequently-asked-questions)
```

Comment:
I wanted to check these work correctly, but kept getting a 404 when trying to visit this page locally. Anyone know how what may be causing this?

Reply:
Actually, that's not what's causing the 404 - unsure about that - but these links still need to be corrected.

Reply:
You might need to run `gatsby clean` and `yarn` again to get rid of cached stuff.

For creating the heading links, I copy them from the links after running locally.

Reply:
I figured out the 404 - it's a typo in a markdown table way below. <atr/> needed to be replaced with <tr/>, IIRC